### ISSUE TITLE
NVDA is sometimes unable to interact at all with menus

### ISSUE THREAD

**Issue by Neurrone** [nvaccess/nvda#11982]
### Steps to reproduce:

This seems to happen randomly, about once a week for me.

### Actual behavior:

When this bug occurs, I notice the following:

1. On the desktop, pressing the applications key when focus is on an item does bring up the context menu, but keyboard focus fails  to move to any of the menu items. This occurs even without NVDA running, so I suspect a bug in Windows itself. Unfortunately, I've had a 0% success rate in reporting bugs to Microsoft, so I hope that it can get through if someone else manages to get their attention.
2. When this bug happens, pressing NVDA + n causes the NVDA menu to occur but again, focus does not move to the menu items. NVDA reacts very badly - pressing any subsequent keyboard commands don't cause anything to happen, similar to a freeze. The only way to recover from the menu being stuck on open and loss of keyboard functionality is to restart NVDA with ctrl+alt+n.

This does not occur with all menus - e.g, pressing the applications key works fine in firefox when this bug occurs for windows menus and the NVDA menu.

The only proper workaround is to restart the computer to have normal functionality restored.

### Expected behavior:

Menus should work properly with the keyboard.

#### NVDA installed/portable/running from source:
Installed

#### NVDA version:
I'll fill this in soon, latest alpha. I cant press NVDA+n as it will cause NVDA to hang.

#### Windows version:
Windows 10 2004

#### Does the issue still occur after restarting your computer?
No

#### Have you tried any other versions of NVDA? If so, please report their behaviors.
Yes, I've seen this for a long time

#### If addons are disabled, is your problem still occuring?
Unknown

#### Did you try to run the COM registry fixing tool in NVDA menu / tools?
No

**Comment by Brian1Gaff** [comment]
This little gem has been around for years, as far back as windows 7. I do 
find logging off the user and then back on can fix it for a while. It is 
some weird lack of interaction with windows and screenreaders, but seems a 
little worse on NVDA than on Jaws.


As you say a reboot can fix it.
Other funnies of a similar nature are when the screenreader thinks the 
current active window goes behind another and reads a menu bar as far as the 
edge of the other window only. Windows has forgotten to let nvda know proper 
info about the screen. I usually find that the best fix for this is to take 
the machine  back to the morning when it was last working or use errunt to 
substitute the old registry files then reboot.
 Corruption of registry is the issue here., that is why it often only 
affects one piece of software.
 Brian

bglists@blueyonder.co.uk
Sent via blueyonder.
Please address personal E-mail to:-
briang1@blueyonder.co.uk, putting 'Brian Gaff'
in the display name field.
Newsgroup monitored: alt.comp.blind-users
----- Original Message -----

**Comment by bhavyashah** [comment]
@Neurrone  When you experience this issue, what happens when you press keyboard shortcuts involving the NVDA key and let's say an alphabet? E.g. NVDA+T or NVDA+B. If NVDA just speaks the alphabet and fails to recognize the press of the modifier key, then that is a very, very long-standing issue that I suspect has been documented in previous tickets.

**Comment by Adriani90** [comment]
I can only reproduce the 1. part of this issue, but this is a windows design bug that is none of the menu items is focused when opening a context menu. NVDA cannot do anything about it.

For the other part of the issue I cannot reproduce with NVDA 2023.1 on Windows 10 or 11. Since we didn't get any updates from the initial author for more than one year, I am closing this issue as abandoned.

**Comment by Neurrone** [comment]
To confirm, I haven't seen this for a long time, so this isn't affecting me anymore.
